src/docs/framework-integration/angular.md

```diff
@@ -89,18 +134,31 @@ import { angularOutputTarget } from '@stencil/angular-output-target';
 export const config: Config = {
   namespace: 'stencil-library',
   outputTargets: [
+    // By default, the generated proxy components will
+    // leverage the output from the `dist` target, so we
+    // need to explicitly define that output alongside the
+    // Angular target
+    {
+      type: 'dist',
+    },
     angularOutputTarget({
-      componentCorePackage: 'your-stencil-library-package-name',
+      componentCorePackage: 'stencil-library',
       directivesProxyFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/components.ts',
       directivesArrayFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/index.ts',
     }),
   ],
 };
 ```
 
-> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
+> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the
+> file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
 
-> The `directivesArrayFile` is the relative path to the file that will be generated with a constant of all the Angular component wrappers. This constant can be used to easily declare and export all the wrappers.
+> The `directivesArrayFile` is the relative path to the file that will be generated with a constant of all the Angular component wrappers. This
+> constant can be used to easily declare and export all the wrappers.
+
+> The `componentCorePackage` should match the `name` field in your Stencil project's `package.json`
+
+See the [API section below](#api) for details on each of the output target's options.
```

Comment:
I have followed the steps recommended I believe exactly, but when I get to the `npm run build` below I'm getting a bunch of typescript errors preventing me from proceeding :/

<img width="1019" alt="Screenshot 2023-01-11 at 10 49 58 AM" src="https://user-images.githubusercontent.com/6207644/211852041-e3b211e7-9b9b-4ad2-8bfa-af4d6f44f97d.png">

Reply:
here's the steps as I've been running them: 

https://gist.github.com/alicewriteswrongs/63533667b566925f2133dd50c66714d8

Reply:
I actually ran into this once too. I think there was something weird going on with Jest peer deps. I tweaked the steps a bit so that shouldn't be an issue (at least for the purpose of this guide)

Reply:
ok so I just updated the gist above (https://gist.github.com/alicewriteswrongs/63533667b566925f2133dd50c66714d8) with the steps in the guide. I'm still getting the same typescript error when I get to `npm run build` though :/

Is there a lerna related thing I'm not getting right here possible?

Reply:
Possibly. I wonder if it's related to `lerna add` I haven't been using those lerna commands since I think they're recommending moving away from that and using [package dependency management](https://lerna.js.org/docs/getting-started#package-dependency-management) which just takes advantage of npm/yarn workspaces. Not sure what `lerna add` really does under-the-hood if that is somehow causing dependency issues 🤷‍♂️

Reply:
I don't think it's related to `lerna add` because I did the steps manually and replaced `lerna add` (where I have it in that script) with an edit to `packages/angular-workspace/packages/component-library/package.json` and I saw the same thing. Maybe it's something else about my machine...I'm not sure. I have been deleting all the output and then running through the steps with a clean slate each time though.

Reply:
Oh wait. This is an issue when building the Stencil project? I thought this was happening when building the Angular lib. I'll run through everything again and see if I hit this, but don't recall having an issue at that step

Reply:
yeah exactly, when I get to the `npm run build` I get a type issue, it looks to me possibly like conflicting typedefs for Jest are being pulled in but I'm not sure

Reply:
Yep. It's a typedef conflict between Jest (which Stencil uses) and Jasmine (from Angular). Removing either solves the build problem. I'll add a step for that in the guide. Think I'll recommend removing Jasmine cuz Angular can work with Jest, but won't go beyond that since it isn't really the scope of the guide (we're not trying to completely architect a project)